### PR TITLE
EZP-27434 proper index.md file for the Tutorial steps

### PR DESCRIPTION
**Pull request by mere-teresa**
The index.md file was initially just a list of links, therefore it was not very useful.
I put back the old index page "Creating a Tweet Fieldtype" content, from the Confluence page. 
I re-arranged everything in 1 commit only.

### CODE REVIEW

**Comment by mere-teresa** [comment]
Ok I will resolve conflicts, then do the PR.

File: docs/tutorial/index.md [hunk 1]
```diff
@@ -1,12 +1,68 @@
...
+
+## Intended audience
+
+This tutorial is aimed at developers who are familiar with eZ Platform and are comfortable with operating in PHP and Symfony2.
+
+## Content of the tutorial
+
+This tutorial will demonstrate how to create a Field Type on the example of a *Tweet* Field Type. It will:
+
+- Accept as input the URL of a tweet (https://twitter.com/&lt;username&gt;/status/&lt;id&gt;)
```
Comment (DominikaK) [inline docs/tutorial/index.md]: You may want to change the link to `(https://twitter.com/{username}/status/{id})` so the brackets don't get misinterpreted.

**Comment by bdunogier** [comment]
Your commits list looks a bit odd, @mere-teresa. Why do they have different authors ?

In addition, this sounds like a lot of commits for one change :)

**Comment by mere-teresa** [comment]
The page "Creating a Tweet Fieldtype" content is now the index page of the Tutorial.

**Pull request merged**